Move to Jigsaw-Python submodule seems to have broken MPAS-Ocean builds

It seems like https://github.com/MPAS-Dev/compass/pull/755 must have broken things on both Anvil and Perlmutter (maybe others).  I tested on Chrysalis but not elsewhere and that seems to have come back to bite me.

Here's what I'm seeing on Perlmutter:
```
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: warning: libpmi.so.0, needed by /opt/cray/pe/mpich/8.1.24/ofi/gnu/9.1/lib/libmpichf90.so, not found (try using -rpath or -rpath-link)
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: warning: libpmi2.so.0, needed by /opt/cray/pe/mpich/8.1.24/ofi/gnu/9.1/lib/libmpichf90.so, not found (try using -rpath or -rpath-link)
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: warning: libcxi.so.1, needed by /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1, not found (try using -rpath or -rpath-link)
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: warning: libcurl.so.4, needed by /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1, not found (try using -rpath or -rpath-link)
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: warning: libjson-c.so.3, needed by /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1, not found (try using -rpath or -rpath-link)
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_clear_wait_obj@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_invalidate_pte_le@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_map_pte@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `curl_multi_strerror'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `curl_easy_getinfo'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_map@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `json_object_array_length'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_pte_transition_sm@LIBCXI_1.4'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `curl_easy_setopt'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/pe/mpich/8.1.24/ofi/gnu/9.1/lib/libmpi_gnu_91.so: undefined reference to `PMI2_Abort'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `json_object_get_boolean'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_get_svc@LIBCXI_1.2'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_read_cntr@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_destroy_pte@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `curl_global_cleanup'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_alloc_lni@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_alloc_ct@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/pe/mpich/8.1.24/ofi/gnu/9.1/lib/libmpi_gnu_91.so: undefined reference to `PMI2_Info_GetJobAttr'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/pe/mpich/8.1.24/ofi/gnu/9.1/lib/libmpi_gnu_91.so: undefined reference to `PMI2_Initialized'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `json_object_get_type'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/pe/mpich/8.1.24/ofi/gnu/9.1/lib/libmpi_gnu_91.so: undefined reference to `PMI2_Job_Spawn'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `json_object_array_get_idx'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_destroy_evtq@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_alloc_cmdq@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_open_device@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `json_object_get_string'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_alloc_evtq@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `curl_easy_init'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_get_svc_rsrc_use@LIBCXI_1.5'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `json_object_get_double'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/pe/mpich/8.1.24/ofi/gnu/9.1/lib/libmpi_gnu_91.so: undefined reference to `PMI2_Finalize'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `json_tokener_parse'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_get_device_list@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `curl_slist_append'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_destroy_cp@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_pte_status@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_unmap@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `json_object_get_int'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `curl_slist_free_all'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_get_wait_obj_fd@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/pe/mpich/8.1.24/ofi/gnu/9.1/lib/libmpi_gnu_91.so: undefined reference to `PMI2_KVS_Put'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_alloc_domain@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_get_svc_list@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_free_device_list@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `curl_multi_init'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `curl_multi_cleanup'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `json_object_get_int64'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_ct_wb_update@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_destroy_lni@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_close_device@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_destroy_cmdq@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/pe/mpich/8.1.24/ofi/gnu/9.1/lib/libmpi_gnu_91.so: undefined reference to `PMI_Barrier'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_destroy_domain@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `curl_easy_cleanup'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_alloc_cp@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `curl_multi_info_read'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_alloc_pte@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/../x86_64-conda-linux-gnu/sysroot/lib64/librt.so.1: undefined reference to `__vdso_clock_gettime@GLIBC_PRIVATE'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/pe/mpich/8.1.24/ofi/gnu/9.1/lib/libmpi_gnu_91.so: undefined reference to `PMI2_Info_GetNodeAttr'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/pe/mpich/8.1.24/ofi/gnu/9.1/lib/libmpi_gnu_91.so: undefined reference to `PMI2_KVS_Get'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_alloc_wait_obj@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/pe/mpich/8.1.24/ofi/gnu/9.1/lib/libmpi_gnu_91.so: undefined reference to `PMI2_KVS_Fence'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/pe/mpich/8.1.24/ofi/gnu/9.1/lib/libmpi_gnu_91.so: undefined reference to `PMI2_Info_PutNodeAttr'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_cmdq_ack_counter@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/pe/mpich/8.1.24/ofi/gnu/9.1/lib/libmpi_gnu_91.so: undefined reference to `PMI2_Job_GetId'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_destroy_ct@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `curl_multi_perform'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `curl_multi_add_handle'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_evtq_adjust_reserved_fc@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/pe/mpich/8.1.24/ofi/gnu/9.1/lib/libmpi_gnu_91.so: undefined reference to `PMI2_Init'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `curl_global_init'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_destroy_wait_obj@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_free_svc_list@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `json_object_object_get_ex'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_unmap_pte@LIBCXI_1.0'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `curl_easy_strerror'
/global/u2/x/xylar/miniforge3/envs/dev_compass_1.2.0-alpha.9/bin/ld: /opt/cray/libfabric/1.15.2.0/lib64/libfabric.so.1: undefined reference to `cxil_get_amo_remap_to_pcie_fadd@LIBCXI_1.3'
collect2: error: ld returned 1 exit status
cc does not support OpenMP - see INSTALL in top-level directory for more information
```

It's also possible that this is related to the recent maintenance but I don't think that's likely.

--- a/conda/bootstrap.py
+++ b/conda/bootstrap.py
@@ -409,11 +409,6 @@ def get_env_vars(machine, compiler, mpilib):
                    f'export I_MPI_F77=ifort\n' \
                    f'export I_MPI_F90=ifort\n'
 
-    if machine == 'anvil':
-        # Anvil seems to need libs from here to build successfully
-        env_vars = f'{env_vars}' \
-                   f'export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/lib64\n'
-
     if machine.startswith('conda'):
         # we're using parallelio so we don't have ADIOS support
         env_vars = \

--- a/conda/bootstrap.py
+++ b/conda/bootstrap.py
@@ -354,8 +354,8 @@ def build_conda_env(env_type, recreate, mpi, conda_mpi, version,
             check_call(commands, logger=logger)
 
             print('Building JIGSAW\n')
+            # add build tools to deployment env, not compass env
             commands = \
-                f'{activate_env} && ' \
                 f'conda install -y cmake cxx-compiler && ' \
                 f'cd {source_path}/jigsaw-python && ' \
                 f'python setup.py build_external'
@@ -371,7 +371,7 @@ def build_conda_env(env_type, recreate, mpi, conda_mpi, version,
 
             t1 = time.time()
             total = t1 - t0
-            message = f'JIGSAW install took {total} s.'
+            message = f'JIGSAW install took {total:.1f} s.'
             if logger is None:
                 print(message)
             else:

